### PR TITLE
GET /api/articles/:article_id/comments #15

### PR DESCRIPTION
**Pull request by jgcav**
GET /api/articles/:article_id/comments #15 - create and pass tests - initially pushed to the branch but made additional commits before submitting pr

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -180,4 +180,50 @@ describe('/api/articles', () => {
...
+              expect.objectContaining({
+                comment_id: expect.any(Number),
+                votes: expect.any(Number),
+                created_at: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String)
+              })
+            )
+          }))
+        })
```
Comment (dav3rid) [inline __tests__/app.test.js:202]: I would include another 200 test to make sure you get an empty array when an article DOES exist - but has no associated comments

File: __tests__/app.test.js [hunk 2]
```diff
@@ -180,4 +180,50 @@ describe('/api/articles', () => {
...
+        .then(({body}) => {
+          expect(body.msg).toBe('No comments found for article ID 3000')
+        })
+  });
+    test('status 404: returns a not found error if valid id is requested but article has no comments', () => {
+        return request(app)
+        .get('/api/articles/7/comments')
+        .expect(404)
+        .then(({body}) => {
+          expect(body.msg).toBe('No comments found for article ID 7')
```
Comment (dav3rid) [inline __tests__/app.test.js]: You have considered the comment I have written above - but I would suggest a 200 👍

**Review by dav3rid — APPROVED** [review]
